app/views/images/show.html.haml

```diff
@@ -0,0 +1,27 @@
+%div{:id => "images"}
+  %p= @image.name
+  = image_tag @image.image.url(:thumb)
+  %br
+  - if current_user == @image.user
+    = link_to "Edit image", edit_image_path(@image)
+    = link_to "Delete image", image_path(@image), method: :delete
+  %br
+  %span.likes_count= @image.likes.count
+  = 'like'.pluralize @image.likes.count
+  %br
+  = link_to image_tag('like.png', :width => 30, :height => 30), image_likes_path(@image), class: 'likes-link', :id => "likebutton"
+  %br
+  - if @image.comments.any?
+    %ul
+      - @image.comments.each do |comment|
+        %div{:id => "comments"}
+          %p
+            %strong
+              = comment.username
+              %strong :
+            = comment.comments
+          - if current_user.username == comment.username
+            = link_to image_tag('edit.png', :width => 20, :height => 20), edit_image_comment_path(comment.id, comment)
+            = link_to image_tag('delete.png', :width => 20, :height => 20), image_comment_path(comment.image_id, comment), method: :delete
+  %div{:id => "addcomment"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace